PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchJobDebuggerRunner.java

```diff
@@ -120,11 +120,28 @@ public GenericDebuggerRunnerSettings createConfigurationData(ConfigurationInfoPr
         return null;
     }
 
+    private void checkSubmissionParameter(SparkSubmissionParameter parameter) throws ExecutionException {
+        if (StringUtils.isEmpty(parameter.getArtifactName())) {
+            throw new ExecutionException("The artifact to submit is not selected, please config it at Run/Debug configuration -> Remotely Run in Cluster.");
```

Comment:
>throw new ExecutionException("The artifact to submit is not selected, please config it at Run/Debug configuration -> Remotely Run in Cluster."); [](start = 12, length = 144)

Question: This will show up in the user UI right? Because this will need user to take some action.